### PR TITLE
10.2.0-M1 release announcement

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: blog/_posts/2020-04-23-akka-http-10.2.0-M1-released.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+Among other things, 10.2.0-M1 contains:
+
+* Improved API's and documentation to improve the experience working with Akka 2.6 and the new Actor API's. Akka HTTP 10.2.0 will keep supporting Akka 2.5.
+* Some new features, including the ability to [attach attributes to requests and responses](https://doc.akka.io/docs/akka-http/current/common/http-model.html#attributes)
+* Improved default configuration, such as [disabling transparent HEAD support by default](https://github.com/akka/akka-http/issues/2088).
+* Various bug fixes, such as around [percent-encoding URI queries](https://github.com/akka/akka-http/pull/3003) and [parsing headers with unicode characters](https://github.com/akka/akka-http/issues/1484)
+* Various improvements to HTTP/2 support, driven by [Akka gRPC](https://doc.akka.io/docs/akka-grpc)
+
+See the Akka HTTP documentation for the [migration guide](https://doc.akka.io/docs/akka-http/current/migration-guide/migration-guide-10.2.x.html) and [more detailed change log](https://doc.akka.io/docs/akka-http/current/release-notes/10.2.x.html)
```
Comment (jrudolph) [inline blog/_posts/2020-04-23-akka-http-10.2.0-M1-released.md:22]: Will we really update the current link? Shouldn't this link to 10.2.0-M1 specifically?

Comment (raboof) [reply blog/_posts/2020-04-23-akka-http-10.2.0-M1-released.md:22]: great point, linking to 10.2.0-M1.

(when 10.2.0 is out we could consider changing the link to point there)

Comment (ennru) [reply blog/_posts/2020-04-23-akka-http-10.2.0-M1-released.md]: Or create the 10.2 link now?

Comment (raboof) [reply blog/_posts/2020-04-23-akka-http-10.2.0-M1-released.md]: good idea, why not

**Pull request merged**